pkgs/by-name/bo/boinctui/package.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, stdenv, fetchFromGitHub, autoreconfHook, expat, ncurses, openssl }:
+
+stdenv.mkDerivation {
+  pname = "boinctui";
+  version = "2.7.1";
```

Comment:
```suggestion
  version = "2.7.1-unstable-2023-12-14";
```

Reply:
Is there an actual reason for doing this though?

While it may not have release tags, it still has proper release commits, indicating a version change. E.g. the used git revision below *exactly* links to the 2.7.1 version release.

Declaring it unstable thus seems kind of wrong IMHO, esp. since not some random git revision is actually used here.

Reply:
Please check [this section](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming) of the documentation:

> If a package is a commit from a repository without a version assigned, then the version attribute should be the latest upstream version preceding that commit, followed by -unstable- and the date of the (fetched) commit. The date must be in "YYYY-MM-DD" format.

As there are no tags/releases on that repository, it should use the indicated format.

Reply:
Alright, thanks for the explanation & reference! I change it. :+1: